src/Model/Term.php

```diff
@@ -290,4 +290,20 @@ public static function populateTagsFromItem(&$item)
 
 		return $return;
 	}
+
+	/**
+	 * Delete all tags from an item
+	 * @param int itemid - choose from which item the tags will be removed
+	 */
+	public static function deleteAllTags($itemid)
+	{
+		$message = Item::selectFirst(['id'], ['id' => $itemid]);
```

Comment:
Please remove this check. If the provided item id doesn't exist in the `term` table, then nothing will happen, but if we want to clean up tags from a non-existing item, they won't be deleted from the `term` table using this method.